docs/tutorials/javascript/send-payments/create-offers.md

```diff
@@ -0,0 +1,1091 @@
+
+This example shows how to:
+
+1. Create currency offers. 
+2. Retrieve active offers.
+3. Match a currency offer to exchange tokens.
+4. Cancel an unsettled offer.
+
+
+[![Offer Create Token Test Harness](/docs/img/module-create-offer.png)](/docs/img/module-create-offer.png)
+
+Download and expand the [Quickstart Samples](https://github.com/XRPLF/xrpl-dev-portal/tree/master/_code-samples/quickstart/js/)<!-- {.github-code-download} --> archive.
+
+**Note:** Without the Quickstart Samples, you will not be able to try the examples that follow. 
+
+## Usage
+<!-- 
+<div align="center">
+<iframe width="560" height="315" src="https://www.youtube.com/embed/YnPccLPa0hc?si=U6uQNXzNRp5G_pri" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+</div>
+-->
+To get test accounts:
+
+1. Open `3a.CreateOffer.html` in a browser
+2. Choose **Testnet** or **Devnet**.
+3. Enter an **Account Name** for the left column. For example, _Standby_.
+4. Click **Get New Account** on the left.
+5. Enter an **Account Name** for the right column. For example, _Operational_.
+5. Click **Get New Account** on the right.
+6. Copy and paste the **Seeds** field in a persistent location, such as a Notepad, so that you can reuse the accounts after reloading the form.
+
+[![Created Standby and Operational Accounts](/docs/img/module-create-offer-get-accounts.png)](/docs/img/module-create-offer-get-accounts.png)
+
+You can create and match offers from either account.
+
+## Create Offer
+
+To create an offer to exchange XRP for an issued currency on the Standby (left) side:
+
+1. Enter _XRP_ as the Taker Pays **Currency**.
+2. Enter the Taker Pays **Value** in drops. For example, _50000000_.
+3. Enter the Taker Gets **Currency**. For example, _USD_.
+4. Copy the left account (_Standby_) value to the Taker Gets **Issuer** field.
+5. Enter the Taker Gets **Value**. For example, _50_.
+6. Click **Create Offer**.
+
+[![Created an offer for XRP and USD](/docs/img/module-create-offer-xrp-for-usd.png)](/docs/img/module-create-offer-xrp-for-usd.png)
+
+To create a complementary offer on the Operational (right) side:
+
+1. Enter the Taker Pays **Currency**. For example, _USD_.
+2. Copy the left (Standby) side **Account** string into the **Issuer** field.
+3. Enter the Taker Pays **Value**. For example, _50_.
+4. Enter _XRP_ as the Taker Gets **Currency**.
+5. Enter the Taker Gets **Value** in drops. For example, _50000000_.
+6. Click **Create Offer**.
+
+[![Results of matching offers for XRP and USD](/docs/img/module-create-offer-xrp-for-usd2.png)](/docs/img/module-create-offer-xrp-for-usd2.png)
+
+## Get Offers
+
+Click **Get Offers** to get a list of offers issued by the corresponding account.
+
+[![Get outstanding offers for the Standby (left) account](/docs/img/module-create-offer-get-offers.png)](/docs/img/module-create-offer-get-offers.png)
+
+## Cancel Offer
```

Comment:
Maybe reorder this so that an existing offer is on the XRPL when you instruct them to Get Offers and Cancel Offers. Something like:

1. Create Offer
2. Get Offer
3. Cancel Offer
4. Create New Offer
5. Create Corresponding Offer

Reply:
I see what you're saying, but I don't think the reader needs to be hand-held to this degree. We'll see if this is a problem in the field, but once they understand how to create an offer it's better if they're left to experiment, rather than have us spoon feed each step.